version.mk

```diff
@@ -1,3 +1,3 @@
-COLLECTOR_VERSION = 1.1.0
+COLLECTOR_VERSION = 1.0.3
```

Comment:
I think we should leave that at 1.1.0 in `master`
If we make a `1.1` release, we would create a `1.1` branch based on current `master` and then bump it there

Reply:
Yeah, I was confused on that. I've got it now!